module/apmfasthttp/context.go

```diff
@@ -78,10 +77,9 @@ func setResponseContext(ctx *fasthttp.RequestCtx, tx *apm.Transaction, bc *apm.B
 		headers.Set(sk, sv)
 	})
 
+	tx.Result = apmhttp.StatusCodeResult(statusCode)
```

Comment:
Result should still be set for non-sampled transactions. Any particular reason why you changed this method?

Reply:
Done!